### PR TITLE
Updated doctrine/coding-standard to 9.0

### PR DESCRIPTION
**Pull request by driehle**

### CODE REVIEW

**Review by snapshotpl — APPROVED** [review]

File: tests/Assets/Entity/Target.php [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <?php
 
+declare(strict_types=1);
+
 namespace DoctrineORMModuleTest\Assets\Entity;
 
-interface TargetInterface
+interface Target
```
Comment (SenseException) [inline tests/Assets/Entity/Target.php:7]: Is this BC break not something for a major release?

Comment (driehle) [reply tests/Assets/Entity/Target.php:7]: I don't think so, as this is in the unit tests and not in the library itself.

Comment (driehle) [reply tests/Assets/Entity/Target.php:7]: Btw, [semver](https://semver.org/) refers to the "public API" of a project and I don't think unit tests are considered part of the public API. Hence, we should be save changing in the unit tests whatever we want.

Comment (SenseException) [reply tests/Assets/Entity/Target.php:7]: > this is in the unit tests and not in the library itself.

Ah, I missed that it's a test interface. Sorry for the holdup.

**Review by SenseException — APPROVED** [review]

File: tests/Assets/Entity/Target.php (hunk 1 above)
Comment (driehle) [reply tests/Assets/Entity/Target.php:7]: No worries 😊

**Pull request merged**